onnx/common/ir_pb_converter.cc

```diff
@@ -498,6 +510,18 @@ void addAttribute(ONNX_NAMESPACE::NodeProto* n_p, Node* n, Symbol name) {
         encodeGraph(g, v);
       }
       break;
+    case AttributeKind::tp: {
+      attr->set_type(ONNX_NAMESPACE::AttributeProto_AttributeType_TYPE_PROTO);
+      auto tp = attr->mutable_tp();
+//      encodeTypeProto(tp, n->tp(name));
```

Comment:
Comments must be resolved one way or another.

Reply:
@yuslepukhin 
Thanks. I'm trying to find a method or write one to copy or clone onnx::TypeProto. Do you have a suggestion for this part?

Reply:
> 
> 
> @yuslepukhin
> Thanks. I'm trying to find a method or write one to copy or clone onnx::TypeProto. Do you have a suggestion for this part?

The method is called Copy()

Reply:
Thanks!